modules/rostests/kmtests/ntos_mm/MmReservedMapping.c

```diff
@@ -292,16 +326,16 @@ TestMap(
     }
 
     MmFreePagesFromMdl(Mdl);
+    ExFreePool(Mdl);
 }
 
 START_TEST(MmReservedMapping)
 {
     PVOID Mapping;
 
     g_IsPae = ExIsProcessorFeaturePresent(PF_PAE_ENABLED);
-    g_OsVersion = SharedUserData->NtMajorVersion << 8 | SharedUserData->NtMinorVersion;
     g_IsReactOS = *(PULONG)(KI_USER_SHARED_DATA + PAGE_SIZE - sizeof(ULONG)) == 0x8eac705;
-    ok(g_IsReactOS == 1, "Not reactos\n");
+    skip(g_IsReactOS == 1, "Not reactos\n");
```

Comment:
```suggestion
    skip(g_IsReactOS == 1, "Not ReactOS\n");
```
I'm not fully sure about increasing the skip count with that. If we have the `trace()` macro in the kmtest driver, then using that instead may be better.

Reply:
@SergeGautherie No.

@HBelusca We have `trace()`, I'll go ahead and do that instead.

Reply:
Yeah, it was added on e0759a5 by @Doug-Lyons.
As is, it looks like to me as a "local" leftover...